src/lib/php/common-sysconfig.php

```diff
@@ -388,9 +388,18 @@ function Populate_sysconfig()
 
   $variable = "FossDashReportingAPIUrl";
   $fossdashApiUrlPrompt = _('FossDash Endpoint URL');
+  $URLValid = "check_email_address";
```

Comment:
Calling "check_email_address" is not appropriate here (plus this function does not check anything...)
A new function should be added, similar to `check_fossology_url` to validate the URL.
`check_fossology_url` cannot be reused here since if also changes global variables, on top of verifying the URL.

Reply:
Ohh sorry, I will fix it. so I missed placed the name of the function. 
Actually, the function name would be **check_url** [here](https://github.com/Orange-OpenSource/fossology/pull/1/commits/9d741682df78aa728e704a91af2a1a772eeac1f7#diff-4d15aa6b33d5b1940de72490f337234eR647), I have made some required changes to validate URL. I think check_url is currently not used by any validation in common-sysconfig file.